SystemTest.py

```diff
@@ -0,0 +1,47 @@
+import os
+import subprocess
+import time
+from datetime import datetime
+import requests
+import WebAPI
+import Explainer
+from ExplainerClient import WebApiClient
+
+# Start the Web API
+web_api_process = subprocess.Popen(['python', 'WebAPI.py'])
+time.sleep(2)  # Wait for the Web API to start
+
+# Start the Explainer
+explainer_process = subprocess.Popen(['python', 'Explainer.py'])
+
+# Wait for the Explainer to initialize
+time.sleep(2)
+
+# Create a Python API client instance
+api_client = WebApiClient('http://localhost:5000')
+
+try:
+    # Upload a sample presentation
+    presentation_path = 'sample.pptx'  # change to your sample
```

Comment:
put the sample.pptx inside your project so that you'd be able to run the test from every computer and every location.